app/assets/stylesheets/components/_youtube-player.scss

```diff
@@ -39,3 +39,20 @@
     border-radius: 100%;
   }
 }
+
+.govuk-details__summary {
+  color: govuk-colour('white');
+  &:hover {
+    color: govuk-colour('white');
+  }
+}
+
+.govuk-details__text {
```

Comment:
Nickpick could we please also add `border-left-color: govuk-colour('white');`  for `.govuk-details__text` –  looks like the border on the left is white in the mockup, and it looks slightly off as a grey colour against the blue background

Reply:
Thanks Dana! Will do.